crucible-llvm/src/Lang/Crucible/LLVM/Translation.hs

```diff
@@ -187,7 +187,7 @@ buildRegTypeMap m0 bb = foldM stmt m0 (L.bbStmts bb)
 
 
 -- | Generate crucible code for each LLVM statement in turn.
-generateStmts :: (?laxArith :: Bool)
+generateStmts :: (?laxArith :: Bool, ?debugIntrinsics :: Bool)
```

Comment:
Since the option controlling lax arithmetic was already being passed around as an implicit parameter, I adopted the same convention for `debugIntrinsics`. Let me know if you'd prefer something else.

Reply:
I think this is fine.  If we end up with more options like this, we should probably bundle them into a record or something.